components/status-indicator/status-indicator.js

```diff
@@ -0,0 +1,99 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class StatusIndicator extends LitElement {
+
+	static get properties() {
+		return {
+			/**
+			 * State of status indicator to display. Valid values are: 'default', 'success', 'alert', and 'none'.
+			 */
+			state: {
+				type: String,
+				reflect: true
+			},
+			/**
+			 * Text for status indicator.
+			 */
+			text: {
+				type: String,
+				value: null

```

Comment:
`value` isn't a thing in Lit -- you set the default value if required in the constructor. In this case though, I think text can just be left `undefined`.